scripts/linters/warranted_angular_security_bypasses.py

```diff
@@ -33,5 +33,6 @@
     # mechanism. The svg-sanitizer file is going to be a permanent member in
     # this list due to the aforementioned reason.
     'core/templates/services/svg-sanitizer.service.spec.ts',
-    'core/templates/services/svg-sanitizer.service.ts')
+    'core/templates/services/svg-sanitizer.service.ts'
+    )
```

Comment:
If you move this to the end of the previous line, so that you don't make any changes to this file, then I'll cease being a codeowner of this PR and you don't need my review :P

Reply:
done